doc/user/posix.md

```diff
@@ -172,13 +172,11 @@ $ daos container info --path <path to entry point>
 
 ### Enabling Caching
```

Comment:
Shall we call this section Caching instead since caching is enabled by default now?